### ISSUE TITLE
Use consistent case for file names

### ISSUE THREAD

**Issue by Cruikshanks** [DEFRA/water-abstraction-team#30]
> I'm basically writing a lot of waffle to justify fixing something I shouldn't have to 🤬 fix. This is development 101 stuff!

Just spotted it today when working on the projects. The previous team(s) have not used a consistent pattern for file naming.

Most use kebab-case, for example `delete-financial-year-ending.js`.  But we've come across lots that don't follow this convention. This screenshot gives an example.

![Screenshot 2022-05-19 at 17 51 41](https://user-images.githubusercontent.com/1789650/169355440-03188596-e3c8-4561-b8da-5e7e185bf07f.png)

Why does it matter? Because we mainly work on the command line. There, character case and patterns matter. It's why most languages now have the convention of

- all lower case
- no spaces

Whether spaces are denoted using underscore or dashes differs by community. JavaScript seems to use both though kebab-case (vs snake_case) seems to be winning out.

Repos that have been checked and files re-named where necessary:

- [x] water-abstraction-import
- [ ] water-abstraction-permit-repository
- [ ] water-abstraction-reporting
- [ ] water-abstraction-returns
- [ ] water-abstraction-service
- [ ] water-abstraction-tactical-crm
- [ ] water-abstraction-tactical-idm
- [ ] water-abstraction-ui
- [ ] water-abstraction-system

**Comment by Jozzey** [comment]
Repos that have been checked and files re-named where necessary:

- [ ] water-abstraction-import
- [ ] water-abstraction-permit-repository
- [ ] water-abstraction-reporting
- [ ] water-abstraction-returns
- [ ] water-abstraction-service
- [ ] water-abstraction-tactical-crm
- [ ] water-abstraction-tactical-idm
- [ ] water-abstraction-ui
- [ ] water-abstraction-system

**Comment by Cruikshanks** [comment]
When the current dev team first joined Water Abstraction we started creating issues for all the problems we saw. Whilst we were still getting to grips with the service it gave us opportunities to fix things and get stuck into the legacy code.

After a while, we became more confident and were able to get stuck into the critical feature at that time; SROC Supplementary billing. It also proved we could implement new and existing functionality in our [water-abstraction-system repo](https://github.com/DEFRA/water-abstraction-system) and integrate it with the existing apps. This, and the ongoing performance and stability issues with the legacy code have meant the whole delivery team are now onboard with us, over time, migrating _all_ functionality across.

So, as a dev team, we're instead going to focus our efforts on migrating functionality rather than fixing the legacy code. It means issues like this are now redundant as we don't intend to ever fix them.
